CMakeLists.txt

```diff
@@ -155,27 +155,10 @@ endif()
 ### Wave components [WW3]
 ###############################################################################
 if(WW3)
-  if(CMAKE_Platform)
-    if(${CMAKE_Platform} MATCHES "linux*" OR ${CMAKE_Platform} MATCHES "macosx*")
-      if (CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
-        set(WW3_COMP "gnu")
-      elseif (CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
-        set(WW3_COMP "intel")
-      endif()
-    else()
-      set(WW3_COMP ${CMAKE_Platform})
-    endif()
-  endif()
-  message("Build WW3:")
-  message("   run: ${CMAKE_BUILD_TOOL} WW3_PARCOMPN=4 WW3_COMP=${WW3_COMP} ww3_nemslibonly")
-  message("   in:  ${CMAKE_CURRENT_SOURCE_DIR}/WW3/model/esmf")
-  message("")
-
-  add_custom_target(ww3_nems
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/WW3/model/esmf
-    COMMAND ${CMAKE_BUILD_TOOL} WW3_PARCOMPN=4 WW3_COMP=${WW3_COMP} ww3_nemslibonly > ${CMAKE_CURRENT_BINARY_DIR}/ww3_make.log 2>&1)
-
-  set(WW3_LIBS ${CMAKE_CURRENT_SOURCE_DIR}/WW3/model/obj/libww3_multi_esmf.a)
+  # Set options for WW3 build
+  set(SWITCH "multi_esmf" CACHE STRING "Default to switch_multi_esmf")
+  set(MULTI_ESMF ON CACHE BOOL "Enable ESMF for WW3")
```

Comment:
I put that there. I thought because they are CACHE options in WW3 that it should be cache here, but maybe that's not needed.

Reply:
Yes, the value of switch could change.   Also, there will be a different cap so the MULTI_ESMF might be changed to a different option at some point in the near future.

Reply:
It needs to be cache because in WW3 because the option is defined like 

`set(MULTI_ESMF OFF CACHE BOOL "Build ww3_multi_esmf library")`. 

If you just use `set(MULTI_ESMF ON)` then in WW3 the value will be overridden to off. If you set it as cache the value won't be overridden because there is no `FORCE`.

Reply:
Thank you both for checking.